packages/datadog_flutter_plugin/README.md

```diff
@@ -1,39 +1,38 @@
-<p align="center">
-  <img src="dd_logo.png" width="200">
-</p>
+## Overview
 
-# Datadog Flutter Plugin
+Datadog Real User Monitoring (RUM) enables you to visualize and analyze the real-time performance and user journeys of your Flutter application’s individual users.
 
-> A Flutter plugin for interacting with Datadog
+## Platform Support
 
-> ⚠️ This plugin is still in Alpha / Developer Preview. 
+| Android | iOS |  Web | MacOS | Linux | Windows |
+| :-----: | :-: | :---: | :-: | :---: | :----: |
+|   ✅    | ✅  |  🚧   | ❌  |  ❌   |   ❌   |
 
-## Current Datadog SDK Versions
+### iOS
+
+Your iOS Podfile must have `use_frameworks!` (which is true by default in Flutter) and target iOS version >= 11.0.
 
-[//]: # (SDK Table)
```

Comment:
Is there any way we can keep this version mapping table, or does it just not render properly at all on the Docs site?